# 
# Submitted:
#
# Robert Schwebel, 2004-08-12
#
# Error: 
#
# Makefile is trying to compile host tool with cross compiler
#
# Description: 
#
# Makefile is trying to compile host tool with cross compiler
#
# State:
# 
# unknown, not not sent upstream yet
#

diff -urN proftpd-1.2.10rc3/lib/libcap/Makefile proftpd-1.2.10rc3-ptx/lib/libcap/Makefile
--- proftpd-1.2.10rc3/lib/libcap/Makefile	2003-05-15 02:49:13.000000000 +0200
+++ proftpd-1.2.10rc3-ptx/lib/libcap/Makefile	2004-08-12 15:49:11.000000000 +0200
@@ -26,7 +26,7 @@
 all: $(LIBNAME)
 
 _makenames: _makenames.c cap_names.sed
-	$(CC) $(CFLAGS) $(LDFLAGS) $< -o $@
+	$(CC_FOR_BUILD) $< -o $@
 
 cap_names.h: _makenames
 	./_makenames > cap_names.h
